sdk/go/auto/cmd_windows.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build windows
+// +build windows
+
+package auto
+
+import (
+	"os"
+	"syscall"
+)
+
+func interruptProcess(proc *os.Process) error {
```

Comment:
Do we need to attribute https://github.com/iwdgo/sigintwindows here?

Reply:
Yeah, probably just pulling the library in instead would make most sense.